.gitignore

```diff
@@ -9,3 +9,4 @@
 /.tox/
 
 /docs/_build/
+__pycache__
```

Comment:
Can you revert this change? Please only include on-topic changes. You can add `__pycache__` to your global gitignore, this'll reduce duplication of such ignores.